tmt/base.py

```diff
@@ -364,7 +365,11 @@ def lint(self):
 
         # Check for possible test case relevancy rules
         filename = self.node.sources[-1]
```

Comment:
Hm, this is a problem - Duplicate can be created by parent node. E.g.
```
==> main.fmf <==
test: y
test: y

==> x.fmf <==
a: b
```
And as such 
```
$ tmt tests lint

/x
pass test script must be defined
pass directory path must be absolute
pass directory path must exist
warn summary is very useful for quick inspection
```
However when I remove `x.fmf` file:
```
/
pass test script must be defined
pass directory path must be absolute
pass directory path must exist
warn summary is very useful for quick inspection
fail fmf metadata must not contain duplicate keys
```